docs/reference/commandline/ps.md

```diff
@@ -59,7 +99,66 @@ The currently supported filters are:
     106ea823fe4e        fedora:latest     /bin/sh -c 'bash -l'   2 weeks ago         Exited (0) 2 weeks ago                              determined_albattani
     48ee228c9464        fedora:20         bash                   2 weeks ago         Exited (0) 2 weeks ago                              tender_torvalds
 
-This shows all the containers that have exited with status of '0'
+This shows all the containers that have exited successfully, i.e., with exited code of '0'.
+
+#### Status
+
+The `status` filter allows you to list containers filtered on the status of the container. You can filter using `created`, `restarting`, `running`, `paused` and `exited`.
```

Comment:
The `status` filter matches containers by status. You can filter using `created`, `restarting`, `running`, `paused` and `exited`.  For example, to filter for `running` containers: